sdk/core/Azure.Core.TestFramework/src/Instrumentation/DiagnosticScopeValidatingInterceptor.cs

```diff
@@ -180,6 +181,9 @@ internal static async ValueTask<T> ValidateDiagnosticScope<T>(Func<ValueTask<(T
             {
                 // Remove subscribers before enumerating events.
                 diagnosticListener.Dispose();
+                var skipOverrideProperty = forwardAttribute is not null ? forwardAttribute.GetType().GetProperty("SkipChecks") : null;
```

Comment:
Why not cast this to the attribute type and use the property directly rather than using reflection?

Reply:
Since its internal shared code this won't work well given different projects can have different versions of the class.